aws/configs/amazon-cloudwatch-agent.json

```diff
@@ -92,43 +43,40 @@
             }
         },
         "append_dimensions": {
-            "ImageId": "${aws:ImageId}",
             "InstanceId": "${aws:InstanceId}",
-            "InstanceType": "${aws:InstanceType}",
             "AutoScalingGroupName": "${aws:AutoScalingGroupName}"
         },
         "aggregation_dimensions": [
-            ["ImageId"],
-            ["InstanceId", "InstanceType"],
             []
-        ],
-        "force_flush_interval": 30
+        ]
     },
     "logs": {
         "logs_collected": {
             "files": {
                 "collect_list": [
                     {
                         "file_path": "/var/log/spacelift/error.log",
-                        "log_group_name": "spacelift-errors.log",
+                        "log_group_name": "/spacelift-worker-image/spacelift-errors.log",
                         "log_stream_name": "{instance_id}-{ip_address}",
-                        "timezone": "UTC"
+                        "timezone": "UTC",
+                        "retention_in_days": 7
                     },
                     {
                         "file_path": "/var/log/spacelift/info.log",
-                        "log_group_name": "spacelift-info.log",
+                        "log_group_name": "/spacelift-worker-image/spacelift-info.log",
                         "log_stream_name": "{instance_id}-{ip_address}",
-                        "timezone": "UTC"
+                        "timezone": "UTC",
+                        "retention_in_days": 7
```

Comment:
Is it safe to add this? According to the docs, adding `retention_in_days` to an existing log group will cause any logs older than that to be deleted.

Reply:
Well, the log group names are updated in this PR, so existing logs to the old log groups will be orphaned and not affected by data loss.